marketplace/app/v0/transformation.py

```diff
@@ -10,55 +10,96 @@ class MarketPlaceTransformationApp(_MarketPlaceAppBase):
+        return transformation.TransformationCreateResponse.parse_obj(
+            json.loads(self._client.post("newTransformation", json=new_transformation))
         )
 
     @check_capability_availability
     def get_transformation(
-        self, transformation_id: TransformationModel.TransformationId
-    ) -> TransformationModel.TransformationModel:
-        params = {"transformation_id": transformation_id}
-        return TransformationModel.TransformationModel.parse_obj(
-            json.loads(self._client.get("getTransformation", params=params))
+        self, transformation_id: transformation.TransformationId
+    ) -> transformation.TransformationModel:
+        return transformation.TransformationModel.parse_obj(
+            json.loads(
+                self._client.get(
+                    "getTransformation", params={"transformation_id": transformation_id}
+                )
+            )
         )
 
     @check_capability_availability
-    def delete_transformation(
-        self, transformation_id: TransformationModel.TransformationId
-    ):
-        params = {"transformation_id": transformation_id}
-        return self._client.delete("deleteTransformation", params=params)
+    def delete_transformation(self, transformation_id: transformation.TransformationId):
+        return self._client.delete(
+            "deleteTransformation", params={"transformation_id": transformation_id}
+        )
 
-    # TODO: check request type (in standard app api its patch)
-    @check_capability_availability
-    def update_transformation(
+    @check_capability_availability("update_transformation")
+    def __update_transformation(
         self,
-        transformation_id: TransformationModel.TransformationId,
-        update: TransformationModel.TransformationUpdateModel,
-    ) -> TransformationModel.TransformationUpdateResponse:
-        params = {"transformation_id": transformation_id}
-        return TransformationModel.TransformationUpdateResponse.parse_obj(
+        transformation_id: transformation.TransformationId,
+        update: transformation.TransformationUpdateModel,
+    ) -> transformation.TransformationUpdateResponse:
+        return transformation.TransformationUpdateResponse.parse_obj(
             json.loads(
-                self._client.put("updateTransformation", params=params, json=update)
+                self._client.patch(
+                    "updateTransformation",
+                    params={"transformation_id": transformation_id},
+                    json=update,
+                )
             )
         )
 
+    def start_transformation(
+        self, transformation_id: transformation.TransformationId
+    ) -> transformation.TransformationStateResponse:
+        update: transformation.TransformationUpdateModel = (
+            transformation.TransformationUpdateModel.parse_obj(
+                {"state": transformation.TransformationState.RUNNING}
+            )
+        )
```

Comment:
This could be simplified to
```suggestion
        update = transformation.TransformationUpdateModel(
            state=transformation.TransformationState.RUNNING)
```